code/game/machinery/computer/cloning.dm

```diff
@@ -324,11 +324,17 @@
 					qdel(C)
 					menu = 1
 				else
-
-					var/mob/selected = find_dead_player("[C.ckey]")
-					selected << 'sound/machines/chime.ogg'	//probably not the best sound but I think it's reasonable
-					var/answer = alert(selected,"Do you want to return to life?","Cloning","Yes","No")
-					if(answer != "No" && pod.growclone(C))
+					var/cloning
+					if(config.use_cortical_stacks)
+						cloning = 1
+						pod.growclone(C)
+					else
+						var/mob/selected = find_dead_player("[C.ckey]")
+						selected << 'sound/machines/chime.ogg'	//probably not the best sound but I think it's reasonable
```

Comment:
I'm not touching anything that isn't explicitly required for the mechanical changes, sorry.